ts/src/bitget.ts

```diff
@@ -3198,8 +3198,8 @@ export default class bitget extends Exchange {
          */
         await this.loadMarkets ();
         const market = this.market (symbol);
-        let marginMode = undefined;
-        [ marginMode, params ] = this.handleMarginModeAndParams ('createOrder', params);
+        const marginParams = this.handleMarginModeAndParams ('createOrder', params);
```

Comment:
@Dan-krm we have to be careful with this, we're omitting `marginMode` here but `createOrderrequest`needed it for spot-margin